elk/argparsers.py

```diff
@@ -210,3 +210,41 @@ def add_train_args(parser: ArgumentParser):
         ),
     )
     return parser
+
+
+def get_evaluate_parser():
+    parser = ArgumentParser(add_help=False)
+    add_eval_args(parser)
+    return parser
+
+
+def add_eval_args(parser):
+    parser.add_argument(
+        "name",
+        type=str,
+        help="Name of the experiment containing" "the reporters you want to evaluate.",
```

Comment:
Same problem here, and a few times below? Looks like you chose to break the literals into pieces much shorter than 88 characters, and then `black` came in and put them on the same line